content/maintainers-guide/releasing-prestashop/minor/feature-freeze-process.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Feature freeze process
+weight: 10
+---
+
+# Feature freeze process
+
+On the day of feature freeze (and the days after, this process can take multiple days).
+
+## 1. Communicate
+
+Starting this step means that the Development phase of this release is over. 
+
+Before you go further, make sure to **tick the "Development" box** in the Release Tracker GitHub issue (there is one per version, see the [8.1.0 example][release-tracker-issue]).
+
+## 2. Create the build branch
+
+* Create a git branch on the Core from `develop` branch.
+
+It will carry the work to be done until the final release. In this process it will be called "the build branch".
+
+It is named after the target release, for example:
+- if the target version is PrestaShop 8.1.0, branch name should be `8.1.x`
+- if the target version is PrestaShop 9.2.0, branch name should be `9.2.x`
+
+{{% notice note %}}
+This is the branch where all future patch versions for this minor version will be developed on (hence the .x at the end).
+{{% /notice %}}
+
+Push this branch to the upstream Core repository.
+
+{{% notice warning %}}
+Write access to GitHub Core repository is needed to push branches.
+{{% /notice %}}
+
+All actions below that request to submit a Pull Request must be done against this new branch.
+
+### Register the new branch
+
+Following the creation of this new branch, update the following files to acknowledge the new branch:
+- [The Nightly builds][nightly-build]
+- [The Pull Request template][pr-template]
+
+## 3. Create the new version in Distribution API
+
+We don't know how to do it yet :D :D :D
```

Comment:
🤣

Reply:
We will know how once we know who :walking:

Reply:
Distribution API has been built on the following concept "Release X is acknowledge by the API when it is released" so it works OK for patch versions but for minor versions where you have a beta period, it fails. We need Distribution API to provide the information for 8.1.0 _before_ 8.1.0 is released not after.

No worries, Distribution API is young and it was expected we would miss some scenarii: it just means there is more to develop